go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/go-mesh/mesher
+
+require (
+	github.com/envoyproxy/go-control-plane v0.5.0
+	github.com/go-chassis/go-chassis v0.8.2
+	github.com/gogo/protobuf v1.1.1
+	google.golang.org/grpc v1.14.0
+	k8s.io/apimachinery v0.0.0-20180904031649-e386b2658ed20923da8cc9250e552f082899a1ee
```

Comment:
unfortunately, need write down replace statement for chinese developer behind the wall


replace (
	github.com/kubernetes/client-go => ../k8s.io/client-go
	golang.org/x/crypto v0.0.0-20180820150726-614d502a4dac => github.com/golang/crypto v0.0.0-20180820150726-614d502a4dac
	golang.org/x/net v0.0.0-20180824152047-4bcd98cce591 => github.com/golang/net v0.0.0-20180824152047-4bcd98cce591
	golang.org/x/sys v0.0.0-20180824143301-4910a1d54f87 => github.com/golang/sys v0.0.0-20180824143301-4910a1d54f87
	golang.org/x/text v0.3.0 => github.com/golang/text v0.3.0
	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 => github.com/golang/time v0.0.0-20180412165947-fbb02b2291d2
	google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8 => github.com/google/go-genproto v0.0.0-20180817151627-c66870c02cf8
	google.golang.org/grpc v1.14.0 => github.com/grpc/grpc-go v1.14.0
)

Reply:
I'll handle this